cmd/check-gh-automation/main.go

```diff
@@ -164,6 +165,28 @@ func checkRepos(repos []string, bots []string, appName string, ignore sets.Set[s
 			return nil, fmt.Errorf("error obtaining repository from github: %s/%s: %w", org, repo, err)
 		}
 
+		configData, found := config.DataWithInfo{}, false
+		if configs.Prow != nil {
+			configData, found = configs.CiOperator[fmt.Sprintf("%s-%s-master.yaml", org, repo)]
```

Comment:
I guess you could replace `fmt.Sprintf("%s-%s-master.yaml", org, repo)` with `(&api.Metadata{Org: org, Repo: repo, Branch: "master"}).Basename()`